src/pages/Home.vue

```diff
@@ -3,8 +3,8 @@
     <WelcomeSection />
     <WhatIsHK />
     <PreRegistration />
-    <SponsorsPartners />
```

Comment:
Revert this change

Reply:
Sponsors & Partners section is below Venue section

Reply:
But done